CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
-0.4.1 (Next)
+0.4.2 (Next)
 ============
 
 * Your contribution here.
 
+0.4.1 (10/23/2015)
```

Comment:
You can amend this via git commit --amend and force push your topic branch and it will update here.